contracts/strategies/strategy-joe-farm-base.sol

```diff
@@ -107,4 +107,16 @@ abstract contract StrategyJoeFarmBase is StrategyBase {
             now.add(60)
         );
     }
+
+    function _takeFeeJoeToSnob(uint256 _keepJOE) internal {
```

Comment:
`  
 path = new address[joe, wavax, snob];
IERC20(joe).safeApprove(joeRouter, 0);
IERC20(joe).safeApprove(joeRouter, _keepJOE);
     `

Reply:
Can you provide more context here? The only difference between your code snippet and what is written is that path is defined earlier. Why would this matter?